CHANGELOG

```diff
@@ -1,3 +1,14 @@
+snowplow-media-player 0.4.2 (2023-06-27)
```

Comment:
Fix version and date, unless plan is to release tomorrow?
```suggestion
snowplow-media-player 0.5.1 (2023-06-27)
```

Reply:
Plan is to release tomorrow. I'll blame the version number on covid though....